src/main/resources/react4xp/_entries/StatisticFigures.tsx

```diff
@@ -0,0 +1,173 @@
+  const [checkOverflow, setCheckOverflow] = useState(false)
+
+  useEffect(() => {
+    if (focusElement && currentElement.current) {
+      const btn = currentElement.current.firstChild?.firstChild as HTMLButtonElement
+      btn.focus()
+    }
+  }, [isHidden])
+
+  function toggleBox() {
+    setIsHidden((prevState) => !prevState)
+  }
+
+  function keyDownToggleBox(e: React.KeyboardEvent<HTMLButtonElement>) {
+    if (e.keyCode === 13 || e.key == 'Enter' || e.keyCode === 32 || e.key == 'Space') {
+      e.preventDefault()
+      setIsHidden((prevState) => !prevState)
+      setFocusElement(!focusElement)
+    }
+  }
+
+  function getButtonBreakpoint() {
+    if (accordions.length > 5) {
+      return ''
+    }
+    return 'd-none'
+  }
+
+  function getButtonText() {
+    if (isHidden) {
+      return (
+        <span>
+          <ChevronDown size='20' className='chevron-icons' /> {showAll}
+        </span>
+      )
+    }
+    return (
+      <span>
+        <ChevronUp size='20' className='chevron-icons' />
+        {showLess}
+      </span>
+    )
+  }
+
+  function renderShowMoreButton() {
+    return (
+      <div className={`row mt-5 hide-show-btn justify-content-center ${getButtonBreakpoint()}`}>
+        <div className='col-auto'>
+          <Button onClick={toggleBox} onKeyDown={(e: React.KeyboardEvent<HTMLButtonElement>) => keyDownToggleBox(e)}>
+            {getButtonText()}
+          </Button>
+        </div>
+      </div>
+    )
+  }
+
+  function renderStatbankBox() {
+    return (
+      <div className='statbank-box'>
+        <div className='content'>
+          <div className='icon-wrapper'>
+            <img src={iconStatbankBox} alt='' />
+          </div>
+          <div className='title-text-wrapper'>
+            <a className='title' href={statbankHref} id='statbankLink'>
+              {statbankBoxTitle}
+            </a>
+            <span className='text'>{statbankBoxText}</span>
+          </div>
+          <ArrowRight size={28} className='arrow-icon' aria-hidden='true' />
+        </div>
+      </div>
+    )
+  }
+
+  function getBreakpoint(index: number) {
+    if (isHidden && index > 4) {
+      return 'd-none'
+    }
+    return ''
+  }
+
+  function getFreeTextBreakpoint() {
+    if (!freeText || (isHidden && accordions.length > 5)) {
+      return 'd-none'
+    }
+    return 'mt-5'
+  }
+
+  function renderAccordionBody(accordion: StatisticFiguresData) {
+    if (accordion.contentType === `${props.appName}:table`) {
+      return <Table checkIsOverflowing={checkOverflow} {...(accordion.props as TableProps)} />
+    } else {
+      // Table or figure from content studio (no user input), hence no need to sanitize
+      return <div dangerouslySetInnerHTML={{ __html: accordion.body! }}></div>
+    }
+  }
+
+  const location = window.location
+  const anchor = location && location.hash !== '' ? location.hash.substr(1) : undefined
+
+  return (
+    <React.Fragment>
```

Comment:
We could probably use `<></>` instead of `React.Fragment`, but I see that these are one of the things we do inconsistently 🤔